src/postprocessing.py

```diff
@@ -220,3 +237,11 @@ def drop_small(img, min_size):
 def label(mask):
     labeled, nr_true = ndi.label(mask)
     return labeled
+
+
+def _get_crop_sequence(crop_vertical, crop_horizontal):
```

Comment:
@apyskir this function was also used in the `augmentations.py` just have it as utils function and use in both places

Reply:
@jakubczakon OK, so either I put it in `utils.py`, or put `self.get_pad` inside this function in `augmentations.py`. I prefer the former.

Reply:
@apyskir me too. You could also have cake and eat it by creating a private method there that calls _get_pad first and then calls get_crop_sequence function

Reply:
@jakubczakon I like cakes. Let's do it.